src/test/java/org/alliancegenome/curation_api/AgmAgmAssociationBulkUploadITCase.java

```diff
@@ -48,84 +49,72 @@ private void loadRequiredEntities() {
 		agmSubject = createAffectedGenomicModel(agmSubjectCurie, "test name subject", "NCBITaxon:7955", "fish", false, dataProvider);
 		agmObject = createAffectedGenomicModel(agmObjectCurie, "test name object", "NCBITaxon:7955", "fish", false, dataProvider);
 	}
-	
+
 	@Test
 	@Order(1)
 	public void agmAgmAssociationBulkUploadCheckFields() throws Exception {
 		loadRequiredEntities();
-		
+
 		checkSuccessfulBulkLoad(agmAgmAssociationBulkPostEndpoint, agmAgmAssociationTestFilePath + "AF_01_all_fields.json");
-	
+
 		RestAssured.given().
 			when().
 			get(agmAgmAssociationGetEndpoint + "?agmSubjectId=" + agmSubject.getId() + "&relationName=" + relationName + "&agmObjectId=" + agmObject.getId()).
 			then().
 			statusCode(200).
 			body("entity.relation.name", is(relationName)).
-			body("entity.agmAssociationObject.primaryExternalId", is(agmObjectCurie)).
 			body("entity.agmAssociationSubject.primaryExternalId", is(agmSubjectCurie)).
 			body("entity.internal", is(false)).
 			body("entity.obsolete", is(false)).
 			body("entity.createdBy.uniqueId", is("AGMTEST:Person0001")).
 			body("entity.updatedBy.uniqueId", is("AGMTEST:Person0002")).
 			body("entity.dateCreated", is(OffsetDateTime.parse("2022-03-09T22:10:12+00:00").toString())).
 			body("entity.dateUpdated", is(OffsetDateTime.parse("2022-03-10T22:10:12+00:00").toString()));
-		
+
 		RestAssured.given().
 			when().
 			get(agmGetEndpoint + agmSubjectCurie).
 			then().
 			statusCode(200).
 			body("entity.agmAgmAssociations", hasSize(1)).
 			body("entity.agmAgmAssociations[0].relation.name", is(relationName)).
-			body("entity.agmAgmAssociations[0].agmAssociationSubject.primaryExternalId", is(agmSubjectCurie)).
-			body("entity.agmAgmAssociations[0].agmAssociationSubject", not(hasKey("agmAssociationObject")));
-
-		RestAssured.given().
-			when().
-			get(agmGetEndpoint + agmObjectCurie).
-			then().
-			statusCode(200).
-			body("entity.agmAgmObjectAssociations", hasSize(1)).
-			body("entity.agmAgmObjectAssociations[0].relation.name", is(relationName)).
-			body("entity.agmAgmObjectAssociations[0].agmAssociationSubject.primaryExternalId", is(agmSubjectCurie)).
-			body("entity.agmAgmObjectAssociations[0].agmAgmAssociationObject", not(hasKey("agmAgmAssociations")));
+			body("entity.agmAgmAssociations[0].agmAssociationSubject.primaryExternalId", is(agmSubjectCurie));
```

Comment:
Probably more informative to test that we can retrieve the `agmAgmAssociationObject` properties here